src/eth/miner/block_miner.rs

```diff
@@ -63,15 +65,15 @@ impl BlockMiner {
                 let mined_log = LogMined {
                     log: mined_log,
                     transaction_hash: input.hash.clone(),
-                    transaction_index,
-                    log_index,
+                    transaction_index: transaction_index.clone(),
```

Comment:
Derive `Copy` for `Index`.

Reply:
I chose not to derive copy and explicitly call `clone()` so we can later optimize those.

Reply:
but since Index is just an integer, I think it's OK to derive Copy now.